src/client/format.ts

```diff
@@ -24,17 +24,16 @@ function format(document: vscode.TextDocument): Promise<vscode.TextEdit[]> {
 			return resolve([vscode.TextEdit.replace(fullDocumentRange(document), stdout)]);
 		}
 
-		// TODO: vscode.workspace.rootPath is deprecated.
 		console.log(`Running ${cmd}...`);
-		childProcess.exec(cmd, { cwd: vscode.workspace.rootPath }, callback);
+		childProcess.exec(cmd, { cwd: workspace.uri.path }, callback);
```

Comment:
`fsPath` should be used instead. `path` contains a "path" part of URI, which is not that thing should be used.

On Windows `workspace.uri.path` is assigned to something like `/c:/Users/esprit/Development/v`, which is not valid path for Windows.

Reply:
can you do that directly  ???